### PR TITLE
Function to serve Symfony projects easily.

### PR DESCRIPTION
**Pull request by sanis**
Just an fuction in aliases added to serve Symfony projects easily.

### CODE REVIEW

File: src/stubs/aliases [hunk 1]
```diff
@@ -32,3 +32,15 @@ function serve-hhvm() {
         echo "  serve-hhvm domain path"
     fi
 }
+
+function serve-symfony2() {
```
Comment (WouterJ) [inline src/stubs/aliases]: please use `symfony` instead of `symfony2`

**Comment by DonPrus** [comment]
We really need Symfony 2 & 3 support at some projects.
Every time I just create a copy of app_dev.php and name it as index.php :)

File: scripts/homestead.rb [hunk 2]
```diff
@@ -141,7 +141,7 @@ def Homestead.configure(config, settings)
       end
 
       if (type == "symfony")
```
Comment (WouterJ) [inline scripts/homestead.rb]: you should change this to `symfony2`, to keep backwards compatibility with people using `symfony2` as type

**Comment by WouterJ** [comment]
@DonPrus there already is Symfony 2/3 support in homestead. See the (not yet merged) documentation here: http://pr-5655-6qmocelev2lwe.eu.platform.sh/cookbook/workflow/homestead.html

**Pull request closed without merging**